doc/stream.markdown

```diff
@@ -1442,7 +1487,7 @@ code) will know when to check again, by calling `stream.read(0)`.  In
 those cases, you *may* call `stream.push('')`.
 
 So far, the only use case for this functionality is in the
-[tls.CryptoStream][] class, which is deprecated in io.js v1.0.  If you
+[tls.CryptoStream][] class, which is deprecated in Node.js/io.js v1.0.  If you
```

Comment:
this is taken from the node docs, but io.js 1.0 was retroactively made node.js 1.0